internal/testdata/validate-lagoon-yml/cronjobs/singleline-cronjobs.lagoon.yml

```
@@ -28,13 +28,6 @@ environments:
         command: |-
           singleline command
 
```

I think theses lines should be added back in.
```suggestion
      - name: block scalar folded clipped 1
        command: >
          singleline
          command
      - name: block scalar folded clipped 2
        command: >
          singleline command
```